snippets/current/src/mediator-setup.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Agent,
+  ConnectionsModule,
+  HttpOutboundTransport,
+  InitConfig,
+  MediationStateChangedEvent,
+  MediatorModule,
+  RoutingEventTypes,
+} from '@aries-framework/core'
+import { IndySdkModule } from '@aries-framework/indy-sdk'
+import { HttpInboundTransport, agentDependencies } from '@aries-framework/node'
+import indySdk from 'indy-sdk'
+
+async function run() {
```

Comment:
We can now just use top level await. So you can remove the `run` method and make everythig top level. This makes that the examples don't have the nesting